portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,8 @@
 
 package com.google.sps.servlets;
 
+import java.util.*;
```

Comment:
Avoid "import java.util.*" for some valuable reasons: https://medium.com/@aayushsharda/why-is-it-better-to-avoid-in-import-statements-in-java-ff9ceaca8aeb

Reply:
Makes sense, thank you!